build.gradle

```diff
@@ -34,7 +37,7 @@ dependencies {
     documentation "org.codehaus.groovy:groovy:$groovyVersion"
     documentation "org.codehaus.groovy:groovy-ant:$groovyVersion"
     documentation "org.codehaus.groovy:groovy-templates:$groovyVersion"
```

Comment:
These should probably be `org.apache.groovy`. As it is now, they get switched out anyway, by the `grails-gradle-plugin`

Reply:
I've pushed all of these changed.